index.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+      @import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300;500;700&display=swap');
+    </style>
+    <link rel="stylesheet" href="./node_modules/normalize.css/normalize.css" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Task-html-and-css</title>
+  </head>
+  <body>
+    <!-- Header section -->
+    <header class="header__container">
+      <nav class="header__nav">
+        <img class="header--logo" src="./images/logo.svg" alt="logo img" />
+        <div>
+          <a class="nav--links" href="#">The Team</a>
+          <a class="nav--links" href="#">Pricing</a>
+          <a class="nav--links" href="#">Features</a>
+        </div>
+      </nav>
+      <section class="header__section">
+        <div class="section--text">
+          <h1 class="section--title">Beautiful Free Nova template</h1>
+          <p class="section-spec">
+            A top notch premium quality template for your next web project.
+          </p>
+          <button class="section--btn">Download FREE!</button>
+        </div>
+        <img
+          class="section--img"
+          src="./images/screen.png"
+          alt="screen with features"
+        />
+      </section>
+    </header>
+    <!-- End of header section -->
+    <!-- Main section -->
+    <main>
+      <!-- Features section -->
+      <section class="features__section">
+        <h1 class="section--heading">Features</h1>
+        <div class="section--items">
+          <!-- Single item div -->
+          <div class="items--single">
+            <div class="icon--container">
+              <span>
+                <img src="./images/responsive.svg" alt="Phone icon image" />
+              </span>
+            </div>
+            <h1 class="items--title">Fully responsive</h1>
+            <p class="items--spec">
+              Looks amazing on any device smartphone, tablet, laptop and
+              desktop.
+            </p>
+          </div>
+          <!-- End of single item div -->
+          <!-- Single item div -->
+          <div class="items--single">
```

Comment:
Jeśli używasz BEM to tak oznaczamy Modyfikator - a on zawsze występuje razem z elementem lub blokiem tj. `items items--single`. Natomiast jeśli ma to być Element to należy pisać tak: `items__single`. Natomiast jeśli taki ma być Element to trzeba w strukturze HTML umieścić gdzieś Block tj. `items`. Proponuję "doczytać" o BEM: https://www.youtube.com/watch?v=0echJVeK0nU